karma.conf.cjs

```diff
@@ -15,7 +16,7 @@ module.exports = function(karma) {
   // we will prefer the unminified build which is easier to browse and works
   // better with source mapping. In other cases, pick the minified build to
   // make sure that the minification process (terser) doesn't break anything.
-  const regex = karma.autoWatch ? /chartjs-plugin-annotation\.js$/ : /chartjs-plugin-annotation\.min\.js$/;
+  const regex = karma.autoWatch ? /chartjs-plugin-annotation\.umd\.js$/ : /chartjs-plugin-annotation\.umd\.min\.js$/;
```

Comment:
I'm not sure if we should repeat this breaking change?

Reply:
But I'm not so expert on these things (JS packaging). Let me know.

Reply:
the esm already has an `esm` part in the name so we are not using `chartjs-plugin-annotation.js` right now with this change as far as I can see so it should be usable instead of adding the umd syntax 🤔

Reply:
It is just a file name.
@etimberg what do you think, should we make the org repositories consistent?

Reply:
@kurkle @LeeLenaleee @etimberg I would suggest do not add UMD in order to avoid a breaking change. When we will define if it must contain UMD, a major version will be bump.
Wdyt?

Reply:
I can not make any change now because I am reinstalling my laptop right now. I am going to do asap

Reply:
I had a look to chartjs-plugin-gradient, what @kurkle did. I'll do the same.

Reply:
done